.editorconfig

```diff
@@ -0,0 +1,18 @@
+root=true
+
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=true
+trim_trailing_whitespace=true
+
+[*.php]
+indent_style=tab
+indent_size=4
+
+[{*.css, *.scss, *.js, *.html, package.json, *.yml}]
+indent_style=space
+indent_size=2
```

Comment:
ok.
the commas thing is not relevant to editorconfig. That should be handled by eslint or similar.
I wrote a [BlogPost about Eslint](https://comsysto.com/blog-post/enforce-javascript-codestyle-for-your-react-application) maybe that helps.

For the indent 4 and php => will do 😄